docs/guides/first-pages/getting-started.md

```diff
@@ -57,57 +69,24 @@ All further pathes are relative to your project root (my-project in this case)
 # Welcome to Rocket
 ```
 
-and start up with `npx rocket start`.
-
-## Add a section
-
-In most cases you will have multiple sections in your website and each of those sections will come with it's own sidebar navigation.
-
-To create a section you need to create a folder with an `index.md`.
+Start up:
 
 ```bash
-mkdir docs/guides
+npx rocket start
```

Comment:
I think we should advocate for adding a 
```
"scripts": {
  "start": "rocket start"
}
```

and then using `npm run start`...

it's probably easier than to explain the difference between npm and npx